### ISSUE TITLE
Task: Add Passenger to Docker container

### ISSUE THREAD

**Issue by icarito** [publiclab/plots2#1599]
This is a prerequisite to putting docker containers in production in accordance to our [update plan](https://github.com/jywarren/web/blob/master/update_plan.md)

**Comment by icarito** [comment]
From https://www.phusionpassenger.com/library/walkthroughs/start/ruby.html
```
You can also use "bundle exec rails server"

If you use Rails, then you can also run bundle exec rails server. As long as you have the above Gemfile entry, that command will start a Passenger-based server.
```
So maybe we don't need any action at all? I'll check staging and see if it runs passenger.

**Comment by icarito** [comment]
Passenger seems installed by the Gem but `rails s` doesn't seem to use it.
```
# pstree -a
bash -c sleep 5 && rails s -p 4000 -b '0.0.0.0'
  `-ruby script/rails s -p 4000 -b 0.0.0.0
      |-{SignalSender}
      |-{ruby-timer-thr}
      `-{ruby}
root@bad5f3eb8b79:/app# passenger-status 
ERROR: Phusion Passenger doesn't seem to be running. If you are sure that it is running, then the causes of this problem could be:

1. You customized the instance registry directory using Apache's PassengerInstanceRegistryDir option, Nginx's passenger_instance_registry_dir option, or Phusion Passenger Standalone's --instance-registry-dir command line argument. If so, please set the environment variable PASSENGER_INSTANCE_REGISTRY_DIR to that directory and run passenger-status again.
2. The instance directory has been removed by an operating system background service. Please set a different instance registry directory using Apache's PassengerInstanceRegistryDir option, Nginx's passenger_instance_registry_dir option, or Phusion Passenger Standalone's --instance-registry-dir command line argument.
```

**Comment by icarito** [comment]
I just submitted a pull request #1601 to do this, should solve this.

**Comment by icarito** [comment]
Although this passed tests and is running in unstable.publiclab.org, it is loggin this every few seconds:
```
web_1   | [ 2017-09-05 05:10:00.0035 70/7fe02bf13700 age/Cor/App/Poo/AnalyticsCollection.cpp:61 ]: ERROR: The '-opid,ppid,%cpu,rss,vsize,pgid,uid,command' command failed                                                                                                                   web_1   |   Backtrace:
web_1   |      in 'void Passenger::ApplicationPool2::Pool::realCollectAnalytics()' (AnalyticsCollection.cpp:183)
web_1   |      in 'static void Passenger::ApplicationPool2::Pool::collectAnalytics(Passenger::ApplicationPool2::PoolPtr)' (AnalyticsCollection.cpp:56)
```
And Solr seems quite slow but I think it may be only warming up on startup? It shouldn't slow the site down so much.

**Comment by icarito** [comment]
Fixed in #1603
